scripts/linqpad/Areas.TransformGoogleSpreadsheetIntoJson.linq

```diff
@@ -47,20 +55,34 @@ public static class Columns
 
 async Task Main()
 {
+	env = "prod"; // dev | prod
```

Comment:
Could you please use dev as environment here? The idea is to change to "prod" only when we need to run the script for production environment.